src/pet.js

```diff
@@ -0,0 +1,111 @@
+const initialFitness = 10;
+const initialHunger = 0;
+
+const hungerGrowUpChange = 5;
+const hungerFeedChange = 3;
+
+const fitnessGrowUpChange = 3;
+const fitnessWalkChange = 4;
+
+const maxFitness = 10;
+const minHunger = 0;
+
+const minFitnessForWalk = 3;
+const minHungerForFeed = 5;
+
+function Pet(name) {
+  this.name = name;
+  this.age = 0;
+  this.hunger = initialHunger;
+  this.fitness = initialFitness;
+  this.children = []
+};
+
+Pet.prototype = {
+  get isAlive() {
+    return this.fitness > 0 && this.hunger < 10 && this.age < 30;
+  }
+};
+
+Pet.prototype.growUp = function() {
+  if (!this.isAlive) {
+    throw new Error('Your pet is no longer alive! :o(')
+  };
+  try {
+  } catch (e) {
+    console.error(e)
+  };
+
+  this.age += 1;
+  this.hunger += hungerGrowUpChange;
+  this.fitness -= fitnessGrowUpChange;
+
+  if (this.fitness > maxFitness) {
+    this.fitness = maxFitness;
+  };
+};
```

Comment:
Theres a few things to look at here. 
First the try catch block: This is not actually dong anything. Normal use of these could be something like this
```
try {
  const myVariable = new yourFunction(param);

  return myVariable.someProperty;
} catch {
  throw new Error('an error message here');
}
```
The idea is you _try_ something you think could fail, and if it does throw an error you then _catch_ it and handle it however is appropriate. With what youve got here, theres nothing in the try block and the error will still be thrown even if you remove the try catch block completely

Second the if statement on line 43: you are checking if the new value for fitness is greater than the maximum, but the assignment on line 41 is reducing the value not increasing it so this if statement never be hit. This means you can remove this as it is redundant